x/gov/keeper/keeper.go

```diff
@@ -71,15 +70,15 @@ func NewKeeper(
 	legacyRouter.Seal()
 
 	return Keeper{
-		storeKey:       key,
-		paramSpace:     paramSpace,
-		authKeeper:     authKeeper,
-		bankKeeper:     bankKeeper,
-		sk:             sk,
-		cdc:            cdc,
-		legacyRouter:   legacyRouter,
-		router:         router,
-		maxMetadataLen: maxMetadataLen,
+		storeKey:     key,
+		paramSpace:   paramSpace,
+		authKeeper:   authKeeper,
+		bankKeeper:   bankKeeper,
+		sk:           sk,
+		cdc:          cdc,
+		legacyRouter: legacyRouter,
+		router:       router,
+		config:       config,
```

Comment:
should we also enforce to use the default config if `config.MaxMetadataLen` is not set?